File: lib/src/main/java/com/darklycoder/download/utils/IOUtil.java
package com.darklycoder.download.utils;

import java.io.Closeable;

public final class IOUtil {

    public static void closeAll(Closeable... closeables) {
        if (null == closeables || closeables.length <= 0) {
            return;
        }

        for (Closeable closeable : closeables) {
            try {
                closeable.close();

            } catch (Exception ignored) {
            }
        }
    }

}

File: lib/src/main/java/com/darklycoder/download/utils/IOUtil.kt
package com.darklycoder.download.utils

import java.io.Closeable

object IOUtil {

    fun closeAll(vararg closeables: Closeable?) {
        for (closeable in closeables) {
            try {
                closeable?.close()

            } catch (ignored: Exception) {
            }
        }
    }

}
